shipyard_blueprints/bigquery/shipyard_bigquery/shipyard/download_to_gcs.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import sys
+
+from shipyard_bigquery import BigQueryClient
+from shipyard_bigquery.utils import exceptions
+from shipyard_templates import ShipyardLogger, ExitCodeException
+import shipyard_bp_utils as shipyard
+
+logger = ShipyardLogger.get_logger()
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--query", dest="query", required=True)
+    parser.add_argument("--service-account", dest="service_account", required=True)
+    parser.add_argument("--bucket-name", dest="bucket_name", required=True)
+    parser.add_argument(
+        "--destination-file-name",
+        dest="destination_file_name",
+        default="output.csv",
+        required=True,
+    )
+    parser.add_argument(
+        "--destination-folder-name",
+        dest="destination_folder_name",
+        default="",
+        required=False,
+    )
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    try:
+        args = get_args()
+        destination_file_name = args.destination_file_name
+        destination_folder_name = (
```

Comment:
Similar simplification here as well

if empty_str returns False

Reply:
What is the simplification here?